core/src/com/unciv/ui/components/tilegroups/TileSetStrings.kt

```diff
@@ -90,18 +106,18 @@ class TileSetStrings(tileSet: String = UncivGame.Current.settings.tileSet, unitS
     fun orFallback(image: String, fallbackImage: TileSetStrings.() -> String): String {
         return if (fallback == null || ImageGetter.imageExists(image))
             image
-        else fallback!!.run(fallbackImage)
+        else fallbackImage.invoke(fallback!!)
```

Comment:
Not sure about this, why this change?
And the onFallback below

Reply:
Fair enough 👍🏿

Reply:
I honestly do not recall, it's possible that the tile and unit fallbacks need redoing